bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -52,60 +52,37 @@ def get_health(self, url: str) -> bool:
             'url': url
         })
 
-    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
-    # https://github.com/iotaledger/iota-sdk/issues/1921
-    def get_info(self) -> NodeInfoWrapper:
-        """Returns general information about the node together with its URL.
-        GET /api/core/v3/info
+    def get_routes(self) -> RoutesResponse:
+        """Returns the available API route groups of the node.
+        GET /api/routes
         """
-        return NodeInfoWrapper.from_dict(self._call_method('getInfo'))
+        return RoutesResponse.from_dict(self._call_method('getRoutes'))
 
-    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
-    # https://github.com/iotaledger/iota-sdk/issues/1921
-    def get_node_info(self, url: str, auth=None) -> InfoResponse:
+    def get_info(self) -> InfoResponse:
```

Comment:
I think NodeInfoResponse with the url should be returned here instead of `get_node_info()`, because for `get_node_info()` the url is provided as parameter so there one knows it, but `get_info()` could be from different nodes

Reply:
that's the part that confuses me a bit ... we don't have that problem with `get_routes`?

Reply:
Theoretically yes, but I think it's just not that interesting, don't know if anyone is even using this endpoint
At least haven't seen anyone using it and nobody asked for it, for nodeinfo we got a request to add the url